Main.java

```diff
@@ -1 +1,39 @@
+package com.company;
 
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Main main = new Main();
+        main.readOrProcessing();
+    }
+
+    private static Comparator<Map.Entry<String, Long>> comparatorForAlphabet() {
+        return (o1, o2) -> {
```

Comment:
describe that on call

Reply:
Ok. If you mean than i should do like : sorted((o1, o2) -> new Comparator().compare(o1, o2)) and create Comparator Class with method compare () 
than i did it
If no, tell me what do you mean